backend/debits/tests.py

```diff
@@ -1,3 +1,215 @@
+from datetime import timedelta
+
+import responses
+
+from django.conf import settings
 from django.test import TestCase
+from django.utils import timezone
+
+from debits.models import Debit
+from debits.providers.easydebit.provider import EasyDebitProvider
+
+
+class TestDebits(TestCase):
```

Comment:
Please split this into two classes `TestDebits` for core functionality and `TestProviderEasyDebit` for provider specific tests.